span.go

```diff
@@ -370,6 +370,9 @@ func (s *Span) End() {
 		s.Context.model.Destination = nil
 		s.Context.model.Service = nil
 
+		// BUG(axw) s.parent.SpanData must be accessed with s.parent.mu held.
+		// Even then, s.parent.SpanData will be nil if ended, so we need to
+		// find an alternative approach.
```

Comment:
Would you mind creating a gh issue for that bug? As I understand this won't be fixed within the same PR.

Reply:
I was intending to fix it in this PR. The other changes could slip to another release, but not this; this causes a panic if you end the parent before the child.